.infrastructure/mysql/mysql-secret.yml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: mysql-secrets
+  namespace: mysql
+type: Opaque
+data:
+    MYSQL_ROOT_PASSWORD: "cm9vdA==" #Base64 "root"
+    MYSQL_DATABASE: "YXBwX2Ri" #Base64 "app_db"
+    MYSQL_USER: "bXlzcWx1c2Vy" #Base64 "mysqluser"
+    MYSQL_PASSWORD: "cm9vdA==" #Base64 "root"
```

Comment:
Using 'root' as a password is a security risk. Consider using a stronger, more complex password and managing it securely with Kubernetes Secrets.